app/models/task.rb

```diff
@@ -0,0 +1,37 @@
+# == Schema Information
+#
+# Table name: tasks
+#
+#  id          :integer          not null, primary key
+#  name        :string
+#  description :text
+#  state       :integer          default(-1)
+#  project_id  :integer
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+# Indexes
+#
+#  index_tasks_on_name  (name) UNIQUE
+#
+
+class Task < ActiveRecord::Base
+  belongs_to :project
+
+  validates :name, presence: true
+  validates :description, presence: true
+
+  after_initialize :set_default_state
+
+  enum state: {
+    todo: -1,
```

Comment:
Any particular reason you chose `-1` to represent the `todo` state? Not saying there is anything wrong with it but it stands out so I thought I'd ask.